docs/posts/flink-quickstart/index.qmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "Ibis goes real-time! Introducing the new Flink backend for Ibis"
+author: "Deepyaman Datta"
+date: "2024-01-29"
+categories:
+    - blog
+    - flink
+    - stream processing
+---
+
+## Introduction
+
+Ibis 8.0.0 marks the official release of the Apache Flink backend for Ibis.
```

Comment:
i'd emphasizing first streaming backend + why this is important

Ibis v8 marks the official release of the first streaming backend for Ibis: the Apache Flink backend. Ibis users can now do data manipulation across both streaming and batch using the same interface. (something like that)